code/src/utest/pcgen/cdom/facet/fact/AgeFacetTest.java

```diff
@@ -48,13 +49,14 @@ protected Integer getItem()
 	@Test
 	public void testItemSetMultGetAge()
 	{
-		assertEquals(0, facet.getAge(id));
+
+		assertEquals(0, (int)facet.get(id));
 		getFacet().set(id, 4);
-		assertEquals(4, facet.getAge(id));
+		assertEquals(4, (int)facet.get(id));
```

Comment:
Do you have a suggestion for how to avoid this? The issue is that assertEquals works both Object and int and thus generates ambiguity.